src/tck/java/io/vertx/codegen/testmodel/JsonMapperTCK.java

```diff
@@ -18,54 +18,54 @@
 @VertxGen
 public interface JsonMapperTCK {
 
-  @Mapper
+  @GenIgnore
   static Integer serializeMyPojoToInteger(MyPojoToInteger value) {
     return value.getA();
   }
 
-  @Mapper
+  @GenIgnore
   static MyPojoToInteger deserializeMyPojoToInteger(Integer value) {
     return new MyPojoToInteger(value);
   }
 
-  @Mapper
+  @GenIgnore
   static JsonObject serializeMyPojoToJsonObject(MyPojoToJsonObject value) {
     return new JsonObject().put("v", value.getV());
   }
 
-  @Mapper
+  @GenIgnore
   static MyPojoToJsonObject deserializeMyPojoToJsonObject(JsonObject value) {
     return new MyPojoToJsonObject(value.getInteger("v"));
   }
 
-  @Mapper
+  @GenIgnore
   static JsonArray serializeMyPojoToJsonArray(MyPojoToJsonArray value) {
     return new JsonArray((List)value.stuff);
   }
 
-  @Mapper
+  @GenIgnore
   static MyPojoToJsonArray deserializeMyPojoToJsonArray(JsonArray value) {
     return new MyPojoToJsonArray(value.stream().map(j -> (Integer)j).collect(Collectors.toList()));
   }
 
-  @Mapper
+  @GenIgnore
   static String serializeZoneDateTime(ZonedDateTime value) {
     return value.toString();
   }
 
-  @Mapper
+  @GenIgnore
   static ZonedDateTime deserializeZoneDateTime(String value) {
     return ZonedDateTime.parse(value);
   }
 
-  @Mapper
-  Function<String, Locale> URL_DESERIALIZER = Locale::new;
+  @GenIgnore
```

Comment:
Why `@GenIgnore`? If those are recognized as mappers, we could ignore them straight from the code

Reply:
Because of the `Function` type is not supported as a constant field of an vertx-gen interface

Reply:
That's what I mean, can we just ignore it, given that we know it's a `Function` that acts as mapper?

Reply:
no we can't that would be another feature

Reply:
Can we open an issue for it? It looks quite important from a user experience POV: think about having an interface annotated with `@ProxyGen` where you add a static mapper for a specific type

Reply:
I disagree, people should have their converters on auxiliary classes and not on `@VertxGen` which would polute the interface. We do it here on `@VertxGen` only because it is convenient.